py_experimenter/experimenter.py

```diff
@@ -29,58 +29,101 @@ def __init__(self,
                  database_name: str = None,
                  name='PyExperimenter'):
         """
-        Initialize PyExperimenter with the configuration file. If table_name or database_name are given they overwrite the
-        values in the configuration file.
-
-        :param config_file: Path to the configuration file. Defaults to 'config/configuration.cfg'.
-        :param table_name: Name of the table in the database. If None, the table name is taken from the configuration file. Defaults to None.
-        :param database_name: Name of the database. If None, the database name is taken from the configuration file. Defaults to None.
+        Initializes the PyExperimenter with the given information.
+
+        :param config_file: The path to the experiment configuration file. Defaults to 'config/configuration.cfg'.
+        :type config_file: str, optional
+        :param credential_path: The path to the database configuration file storing the credentials for the database connection, i.e., host, user and password. Defaults to 'config/database_credentials.cfg'.
```

Comment:
credential_path vs config_file: 
1) inconsistent naming. Both seem to be paths pointing to files and such they should be named similarly. 
2) credential_path seems ambigious to me. How about "database_credential_file_path" and then "config_file_path". Alternatively, we can also just leave out the "path" in the name completely.